### DISCUSSION TITLE
While Deployment of fast api app in versel facing some issue please help with this.

### DISCUSSION THREAD

**Discussion by pramodpm2** [vercel/community#2232] [Bug Report]
Error: Command failed: pip3.9 install --disable-pip-version-check --target . --upgrade -r /vercel/path0/requirements.txt
--
10:54:48.192 | error: subprocess-exited-with-error
10:54:48.192 |  
10:54:48.192 | × python setup.py egg_info did not run successfully.
10:54:48.192 | │ exit code: 1
10:54:48.193 | ╰─> [16 lines of output]
10:54:48.193 | /bin/sh: mysql_config: command not found
10:54:48.193 | /bin/sh: mariadb_config: command not found
10:54:48.193 | /bin/sh: mysql_config: command not found
10:54:48.193 | Traceback (most recent call last):
10:54:48.193 | File "<string>", line 2, in <module>
10:54:48.193 | File "<pip-setuptools-caller>", line 34, in <module>
10:54:48.193 | File "/tmp/pip-install-b4vtypti/mysqlclient_f54356b128cb463fa35d4dbb161bcb53/setup.py", line 15, in <module>
10:54:48.193 | metadata, options = get_config()
10:54:48.193 | File "/tmp/pip-install-b4vtypti/mysqlclient_f54356b128cb463fa35d4dbb161bcb53/setup_posix.py", line 70, in get_config
10:54:48.193 | libs = mysql_config("libs")
10:54:48.193 | File "/tmp/pip-install-b4vtypti/mysqlclient_f54356b128cb463fa35d4dbb161bcb53/setup_posix.py", line 31, in mysql_config
10:54:48.193 | raise OSError("{} not found".format(_mysql_config_path))
10:54:48.194 | OSError: mysql_config not found
10:54:48.194 | mysql_config --version
10:54:48.194 | mariadb_config --version
10:54:48.194 | mysql_config --libs
10:54:48.194 | [end of output]
10:54:48.194 |  
10:54:48.194 | note: This error originates from a subprocess, and is likely not a problem with pip.
10:54:48.194 | error: metadata-generation-failed
10:54:48.194 |  
10:54:48.194 | × Encountered error while generating package metadata.

</body>
</html>

**Comment by parth-codes22** [comment]
Did you manage to resolve it?
